source/_integrations/xiaomi_miio.markdown

```diff
@@ -423,6 +424,37 @@ Switch                  | Description
 Buzzer                  | Turn on/off the buzzer
 Child Lock              | Turn on/off the child lock
 
+### Air Purifier 3C (zhimi.airpurifier.mb4)
+
+This model uses newer MiOT communication protocol.
```

Comment:
```suggestion
```
I don't think this is necessary to point out for our users. Otherwise LGTM :+1:

Reply:
Removed